common/kvstore/tablestore/config.go

```diff
@@ -36,18 +36,27 @@ type Config struct {
 	Schema []string
 	// Optional Prometheus registry for metrics collection. If nil, metrics collection is disabled.
 	MetricsRegistry *prometheus.Registry
+
+	// If true, levelDB will disable seeks compaction. Default is true. This setting is ignored if the base store is
```

Comment:
When this table store was suggested, I made a point of leaky abstraction. This looks getting started.

Reply:
I agree, this is ugly.

Perhaps a better way to handle this sort of custom configuration for different impls is to make the base DB object something that gets passed into the constructor of the wrapper object. Not something for this PR of course, but perhaps something to consider in the future.